pyproject.toml

```diff
@@ -1,3 +1,63 @@
 [build-system]
-requires = ["setuptools >= 46.4.0", "wheel"]
+requires = ["setuptools >= 61.0.0", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "afkak"
+dynamic = ["version"]
+authors = [{name = "Robert Thille"}]
+requires-python = ">=3.6"
+readme = "README.md"
+description = "Twisted Python client for Apache Kafka"
+keywords = ["kafka", "distributed messaging", "txkafka"]
+classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Framework :: Twisted',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: POSIX',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation :: PyPy',
+        'Topic :: Communications',
+        'Topic :: System :: Distributed Computing',
+]
+
+dependencies = [
+        'attrs >= 19.2.0',  # For functioning auto_exc=True.
+        'Twisted >= 18.7.0',  # First release with @inlineCallbacks cancellation.
+]
+maintainers = [
+        {name = "Ciena"},
+        {name = "Mark Vaught"}
+]
+
+
+[project.urls]
+Home-page = "https://github.com/ciena/afkak"
+Documentation = "https://afkak.readthedocs.io/en/latest/"
+Issues = "https://github.com/ciena/afkak/issues"
+Source = "https://github.com/ciena/afkak"
+
+[project.optional-dependencies]
+FastMurmur2 = ["murmurhash2"]
+snappy = ["python-snappy"]
+
+[tool.setuptools.dynamic]
+version = {attr = "afkak.__version__"}
+
+[tool.setuptools.packages]
+find = {}
+
+[tool.black]
+line-length = 120
+target-version = ['py37', 'py38', 'py39']
+src = "afkak"
+skip-string-normalization=true
```

Comment:
I see we have this, which I think is great because being picky about which quotes are used is probably the thing I hate the most about black. But I see quotes were normalized throughout this PR. Is this setting ignored? Or was black run over this before the setting was added?

Reply:
black was run before this setting was being picked up correctly i did a quick test on a new change and verified its working correctly now